packages/babel-register/test/browserify.js

```diff
@@ -1,25 +0,0 @@
-import browserify from "browserify";
-import path from "path";
-import vm from "vm";
-import { fileURLToPath } from "url";
-
-describe("browserify", function () {
-  it("@babel/register may be used without breaking browserify", function () {
```

Comment:
Can you elaborate more on why this test is not needed anymore?

Reply:
The reason why this test was added is that `@babel/core` previously depended on `@babel/register`, and we hope that `@babel/core` can be used in the browser through `browserify`.
Now that it is no longer relied on, `@babel/register` itself does not need to be ensured to run in the browser.

Reply:
Ok thanks! Could we keep it for Babel 7?

Reply:
OK, I'll leave it as a TODO.

Reply:
Can also just use `itBabel7`